src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+div {
```

Comment:
styling div is too general
what if you need to add another block that does not need these styles